CorsixTH/Lua/languages/english.lua

```diff
@@ -493,7 +493,7 @@ tooltip.folders_window = {
   savegames_location = "By default, the Saves directory is alongside the config file and will be used for storing saved games in. Alternatively, you can choose your own by browsing to the directory that you want to use.",
   screenshots_location = "By default, the Screenshots are stored in a folder alongside the config file. Alternatively, you can choose your own by browsing to the directory that you want to use.",
   music_location = "Select a location for your music files by browsing to the directory you want to use (directory must already exist beforehand).",
-  browse_data = "Browse for another location of a Theme Hospital installation (current location: %1%)",
+  browse_data = "Browse for another location for your Theme Hospital installation (current location: %1%)",
```

Comment:
You still didn't capitalize Current location like the others?

Reply:
Oops sorry was doing it in a rush, I'll update that.